### PR TITLE
Add OLS-9 participants to the website

### PR DESCRIPTION
**Pull request by NPDebs**
:+1::tada: First of all, thanks for taking the time to contribute! :tada::+1:

## FOR CONTRIBUTOR

* [x] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [ ] This PR adds a new blog post
* [x] This PR does something else (explain above)

**This PR fixes issue #900.**
Here, the participants of the ninth cohort of OLS have been added to the people page.

**Changes made so far:**

- [ ] OLS-9-Catalyst
   - [x] People
   - [x] [Projects](https://deploy-preview-907--ols-bebatut.netlify.app/openseeds/ols-9-catalyst/projects-participants#projects), with mentees and mentors. (TBD: Add other team leads)
   - [ ] Experts
   - [x] [Facilitators](https://deploy-preview-907--ols-bebatut.netlify.app/openseeds/ols-9-catalyst/#facilitators)
- [ ] OLS-9-DRI
   - [x] People (TBD: Sort Azam's name)
   - [x] [Projects](https://deploy-preview-907--ols-bebatut.netlify.app/openseeds/ols-9-dra/projects-participants#projects), with mentees and mentors.
   - [ ] Experts
   - [ ] Facilitators

[Deploy preview](https://deploy-preview-907--ols-bebatut.netlify.app/openseeds/)...

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [ ] The website builds (a check will fail if not)
* [ ] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [ ] The preview corresponds to the changes described in the Pull Request
* [ ] The code is tidy and passes the linting tests

### CODE REVIEW

**Comment by malvikasharan** [comment]
@NPDebs Just checking if you managed to find OLS-9 DRA track entries for mentors and mentees. I can ping you the csv files if easier.

**Comment by yochannah** [comment]
we chatted - can confirm she's aware of this.

**Comment by NPDebs** [comment]
Thanks both, @malvikasharan & @yochannah.

**Comment by NPDebs** [comment]
We've made significant progress with this PR, and the changes have been documented in the PR description.
@malvikasharan & @Tajuddeen1 - I would appreciate a review from you both. 🙏 

Please note the **changes to be made:**

- OLS-9-Catalyst
   - [Projects](https://deploy-preview-907--ols-bebatut.netlify.app/openseeds/ols-9-catalyst/projects-participants#projects) - Add other team leads (as discussed Taj).
   - Add experts besides Debs.

- OLS-9-DRI
   - Add experts and facilitators.

**Comment by NPDebs** [comment]
Thanks for providing the sheet containing a comprehensive list of participants, @Tajuddeen1. The projects have been updated to include all project team leads. 

Do have a look at [the preview](https://deploy-preview-907--ols-bebatut.netlify.app/openseeds/ols-9-catalyst/projects-participants#projects) when you can. 🙏🏽

**Comment by seunolufemi123** [comment]
Hi @NPDebs, 

Nice work!!!

A little observation here, I am not sure it really counts- the project: "DECIPHERING THE ROLE OF PHOSPHAaTIDYLETHANOLAMINE TRANSFERASE (PEMT) IN GLIOBLASTOMA MULTIFORME CELL INVASION, PROLIFERATION AND AGGRESSION is all writing in capslock". It appears screaming, I think it will make sense if you can convert it to sentence case.

**Comment by NPDebs** [comment]
Thanks, Seun! 
I had the **exact thought** about that project name, but I was skeptical about altering the state of any submission. If it bothered us both, it's worth editing for the sanity of others.

File: _data/openseeds/ols-9-catalyst/projects.yaml [hunk 1]
```diff
@@ -1,3 +1,197 @@
...
+  participants:
+  - monsurat-onabajo
+  - mqnifestkelvin
+  - bilikis-fadeyi
+  mentors:
+  - malloryfreeberg
+  keywords:
+  - Electronic health records
+  - Healthcare system
+- name: DECIPHERING THE ROLE OF PHOSPHAaTIDYLETHANOLAMINE TRANSFERASE (PEMT) IN GLIOBLASTOMA
```
Comment (NPDebs) [inline _data/openseeds/ols-9-catalyst/projects.yaml:150]:
```suggestion
- name: Deciphering the Role of Phosphatidylethanolamine Transferase (PEMT) in Glioblastoma
```

**Comment by malvikasharan** [comment]
Thanks @NPDebs -- this looks like a great progress. There are few things to note that needs to fixed before merging:

- total number of project is 80, shows 97 
image

- Mentor for Simplice is Lessa - currently doesn't show anyone

- Azam's details are not visible
image

- Twitter links are not rendering well as some have added https://x.com which needs to be edited out
image

- Linkedin links are not rendering correctly, with some overflowing the box: 
image

For Catalyst - Seun's project should have Taj also added as a co-mentor

File: _data/openseeds/ols-9-catalyst/projects.yaml [hunk 2]
```diff
@@ -1,3 +1,197 @@
...
+    and continuous learning. This initiative will create a sustainable ecosystem of
+    professionals capable of driving innovation and contributing to global scientific
+    advancements, ultimately bridging the expertise gap and promoting a more inclusive
+    scientific community.
+  participants:
+  - seunolufemi123
+  - emmanuel19-ada
+  - aymd23
+  mentors:
+  - malvikasharan
```
Comment (NPDebs) [inline _data/openseeds/ols-9-catalyst/projects.yaml:37]:
```suggestion
  - malvikasharan
  - tajuddeen1
```

File: _data/openseeds/ols-9-dra/projects.yaml [hunk 3]
```diff
@@ -1,3 +1,1439 @@
...
+  description: En tant que champion de l'EDI, je vise à développer mes compétences
+    pour favoriser la collaboration entre chercheurs et communautés diverses. Je souhaite
+    approfondir ma compréhension des meilleures pratiques en matière d'accès ouvert
+    aux données, afin de rendre la recherche accessible à tous. Enfin, je m'efforce
+    d'explorer des approches innovantes pour engager activement les parties prenantes,
+    garantissant ainsi que leurs voix soient intégrées dans le processus de recherche.
+  participants:
+  - simplice94
+  mentors:
+  - lessamaker
```
Comment (NPDebs) [inline _data/openseeds/ols-9-dra/projects.yaml:941]:
```suggestion
  - lessa-maker
```

**Comment by NPDebs** [comment]
Thanks a lot for the extra pair of eyes, Malvika! 🚀 

- Taj has been added as a co-mentor for BoN.
- Simplice’s mentor, Lessa, was present, but missing a hyphen (i.e lessamaker, not lesser-maker).
- Twitter and Linkedin links have also been updated.

> total number of project is 80, shows 97

This is a bug B.B and I have happily lived with until now. 😅 You'll notice it's the same for every other cohort because there’s a glitch in how the loop runs. If that doesn't bother you terribly, we can work on fixing that in the not-so-distant future.

**Comment by NPDebs** [comment]
For Azam - her id in the `people.yaml` is `that-souns-wonderful.-i-have-just-created-one-account.:`. So, that needs to be corrected in the file and on Civi.

File: _data/people.yaml [hunk 4]
```diff
@@ -9290,6 +11422,28 @@ teresa-m:
   continent: Europe
   longitude: 7.8494005
   latitude: 47.9960901
+that-souns-wonderful.-i-have-just-created-one-account.:
```
Comment (NPDebs) [inline _data/people.yaml:11425]:
```suggestion
azam-dashti:
```

File: _data/openseeds/ols-9-dra/projects.yaml [hunk 5]
```diff
@@ -1,3 +1,1439 @@
...
+  description: First, I aim to learn how to support my teammates and researchers in
+    the Participatory Cultures Lab. This firsthand experience deepens my understanding
+    of how this method works and how inclusive research practices can be integrated
+    into open-access platforms. I have always been interested in knowledge mobilization
+    and how we, as researchers, can facilitate the circulation of knowledge. I am
+    eager to explore strategies for fostering diverse collaborations that promote
+    transparency, inclusivity, and ethical considerations, especially in the context
+    of interdisciplinary and cross-cultural studies and research outcomes.
+  participants:
+  - Azam
```
Comment (NPDebs) [inline _data/openseeds/ols-9-dra/projects.yaml:768]:
```suggestion
  - azam-dashti
```

**Comment by yochannah** [comment]
Okay. It builds, and Malvika looks happy with the content. @bebatut I'm merging, and adding the "tech review needed" tag.

**Review by yochannah — APPROVED** [review]

**Pull request merged**